File: hw/collahuazo/u3/hw23AbstractFactory/AbstractFactoryJavaScritp/controller/winFactory.js
import GUIFactory from './guiFactory.js';
import WinButton from '../model/winButton.js';
import WinMenu from '../model/winMenu.js';

class WinFactory extends GUIFactory {
  createButton() {
    return new WinButton();
  }

  createMenu() {
    return new WinMenu();
  }
}

export default WinFactory;

File: hw/aimacana/u3/hw23AbstracFactory/AbstracFactory/src/main/java/ec/edu/espe/abstracfactory/model/WinFactory.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */

package ec.edu.espe.abstracfactory.model;

/**
 *
 * @author Cristian Palomo, Error 404, @ESPE
 */
public class WinFactory implements GUIFactory{
    
    @Override
    public Button createButton() {
        return new WinButton();
    }

    @Override
    public Menu createMenu() {
        return new WinMenu();
    }
}
